### DISCUSSION TITLE
Silent Mode + Eco Mode sets APU STAPM Limit to 62W?

### DISCUSSION THREAD

**Discussion by xanver999** [seerge/g-helper#2619] [Q&A] [answered]
Running Ghelper on my G14 2023 with Ryzen 7940HS. I set mine to Silent mode(best power efficiency+boost disabled) with Eco mode. While running on AC, HWMonitor reports APU STAPM limit of 35W, which seems fine. But as soon as I unplug it and run on battery the STAPM limit increases to 62W. Is it normal? Or somethings wrong?
![On AC](https://github.com/seerge/g-helper/assets/81725160/5934f395-e77b-47bf-a4a3-f118ae4b919e)
![On Battery](https://github.com/seerge/g-helper/assets/81725160/15680bd9-9e6e-4a46-bcb0-10ebebddc161)

**Comment by seerge** [comment, marked as answer]
@xanver999 hello, this is normal and how manufacturer has programmed this modes for your device

**Reply by xanver999** [reply]
Oh ok, no worries then. Thanks for the quick reply.
